templates/CRM/UF/Page/Group.hlp

```diff
@@ -35,8 +35,8 @@
     {capture assign=configContribURL}{crmURL p='civicrm/admin/contribute' q='reset=1'}{/capture}
     <li>{ts 1=$configContribURL}<strong>CiviContribute</strong> - When you want to collect information from Contributors via online contribution pages, you can create a profile and link it to to your contribution page as a 'custom page element' (<a href='%1'>Configure Online Contribution Pages</a>).{/ts}</li>
     {capture assign=siteRoot}&lt;{ts}site root{/ts}&gt;{/capture}
-    <li>{ts 1=$siteRoot 2='civicrm/profile?reset=1'}<strong>Contact Search and Listings</strong> - When you want to provide a search form and search result listings for some or all of your contacts you can create a profile with the desired search fields and link users to the path for that search form. When creating a link to search and listings profiles, add the corresponding profile ID to the end of your query string using the 'gid' parameter. For example, the link to display a search and listings page for a Profile with ID = 3 would be:{/ts} <em>{$siteRoot}/civicrm/profile?reset=1&amp;gid=3</em>
-        You can limit the search results to a specific group of contacts. EXAMPLE: You only want your organization's staff members to be searchable. You would do this by limiting your profile to only contacts in your 'Staff' group under the profile's Advanced Settings.
+    <li>{ts 1=$siteRoot 2='civicrm/profile?reset=1'}<strong>Contact Search and Listings</strong> - When you want to provide a search form and search result listings for some or all of your contacts you can create a profile with the desired search fields and link users to the path for that search form. When creating a link to search and listings profiles, add the corresponding profile ID to the end of your query string using the 'gid' parameter. For example, the link to display a search and listings page for a Profile with ID = 3 would be:{/ts} <em>{$siteRoot}/civicrm/profile?reset=1&amp;gid=3</em>{/ts}
+      {ts}You can limit the search results to a specific group of contacts. EXAMPLE: You only want your organization's staff members to be searchable. You would do this by limiting your profile to only contacts in your 'Staff' group under the profile's Advanced Settings.{/ts}
     </li>
```

Comment:
Ya I know that was a miss on my part. Actually in order to allow newline I thought better to end-n-start new {ts} tag on newline but that was a flawed attempt. Anyways I will soon update the PR with proper change and also other remaining fixes.